src/js/components/Stats.jsx

```diff
@@ -208,7 +208,16 @@ class Stats extends Component {
                         }}
                         bars="vertical"
                     />
-
+                    <h1>Active users per day</h1>
+                    {this.state.active_users.map(users_per_day =>
+                        <h2>users_per_day.day</h2>
+                        <Chart
```

Comment:
I think this is Whats giving you an error. I'm not too sure how react-charts is supposed to work But I think that the problem is that you are creating multiple charts that hold just one entry. Can't you pass the data like you did above with number of replies?